WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -770,9 +770,13 @@ public void onUploadCanceled(MediaModel media) {
 
     @Override
     public void onUploadError(MediaModel media, MediaStore.MediaError error) {
-        AnalyticsTracker.track(Stat.EDITOR_UPLOAD_MEDIA_FAILED);
         String localMediaId = String.valueOf(media.getId());
 
+        MediaFile mf = FluxCUtils.fromMediaModel(media);
```

Comment:
is there any chance that `media` here comes as a `null` object? I'd add a safety guard here just in case